packages/shorebird_cli/lib/src/validators/shorebird_flutter_validator.dart

```diff
@@ -159,6 +159,7 @@ This can cause unexpected behavior if you are switching between the tools and th
       'flutter',
       ['--version'],
       useVendedFlutter: !checkPathFlutter,
+      runInShell: true,
```

Comment:
Does windows require runInShell (I seem to recall it does)

Reply:
Yep, although seemingly not always. The git commands run find without it. I'm sure I'll learn a lot more about this in the days to come.